LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -1193,9 +1193,22 @@ BriefSummary="Massively reduces weapon noise and the soldier's impact on mission
 TinySummary="Reduces mission infiltration time"
 LootTooltip="This weapon upgrade, when installed, will reduce the time the soldier requires to infiltrate during a mission. It also reduces weapon noise during combat."
 
+; LWOTC needs translation
+
+[CritUpgrade_Bsc X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +<Ability:CRITINCREASE/>%".
```

Comment:
Actually, aren't the full stops supposed to be inside the quotes? Pretty sure they are. Pleas change!

Reply:
🤦‍♂ Fixed